src/test/scala/vct/helper/SimpleProgramGenerator.scala

```diff
@@ -0,0 +1,26 @@
+package vct.helper
+
+import vct.col.ast.Constant._
+import vct.col.ast.{ApplicableContract, Class, FileOrigin, InputOrigin, InstanceMethod, Origin, Program, Statement, TVoid}
```

Comment:
Utility code like this is fine, but it seems pretty specific for each unit test. So maybe this can instead be included in the unit tests class, or in the same file but as a separate object? Then you can also pick better names, for example, instead of generateSimpleInputOrigin, you can just use generateOrigin, since it's usage locally in the unit test class is clear. ("Simple" is a very bad name anyway, imo :) )